entrypoint.sh

```diff
@@ -38,6 +41,40 @@ if [ -z "$DEBUG" ]; then
   trap 'rm -rf -- "$MYTMPDIR"' EXIT
 fi
 
+clean () {
+  echo "=====> cleaning phase"
+  rm -Rf \
+    ./content/* \
+    ./package.json \
+    ./package-lock.json \
+    ./go.sum \
+    ./node_modules \
+    ./resources 
+}
+
+build_indexes() {
+  local repo="$1"
+  local branch="$2"
+  local remote_dir="$3"
+  local local_dir="$4"
+  local version="$5"
+  local latest="$6"
+
+  export REPOSITORY="$repo" BRANCH="$branch" REMOTE_DIR="$remote_dir" LOCAL_DIR="$local_dir"
+
+  if [[ -n $version ]]; then
+    cp "scripts/versioned-reference-index.md" "content/$local_dir/reference/_index.md"
+    export VERSION="$version" LATEST="${latest:-false}"
+    envsubst '${REPOSITORY} ${BRANCH} ${REMOTE_DIR} ${LOCAL_DIR} ${VERSION} ${LATEST}' < "scripts/versioned-reference-section-index.md" > "content/$local_dir/reference/$version/_index.md"
```

Comment:
Shouldn't we use something like:

```suggestion
    cat > "content/$local_dir/reference/$version/_index.md" << EOF
---
cascade:
  version: "${VERSION}"
  latest: ${LATEST}
  docsRepo:
    url: "https://github.com/gatling/${REPOSITORY}/blob/${BRANCH}/${REMOTE_DIR}"
    rel: "${LOCAL_DIR}"
---
EOF
```

Note: no need to export variables with this.

Reply:
Ok, I understand your point.
I'm not a big fan of the `envsubst` command either :stuck_out_tongue: 

So we should find a better name for the "`scripts`" directory, then.

What will be the solution for the left menu?
Perhaps having something that manage all this yaml stuff in the same place (and not in the bash script) can be another solution.

Reply:
Menu yaml is now inferred from the structure of the content

Reply:
`template/indexes`